test/Makefile

```diff
@@ -16,9 +16,9 @@ ifdef CMD_EXE
   RMDIR = rmdir /s /q $(subst /,\,$1)
 else
   EXE :=
-  DEL = $(RM) $1
+  DEL = $(RM) -f $1
```

Comment:
According to the GNU make doc the default value for `RM` is `rm -f`. That's the reason why I introduced `$(RM)` in the first place. Are you sure that the `-f`is actually necessary?